tests/features_and_data/test_distribution.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import os
+
+from src.data.get_data import main as get_data
+
+
+@pytest.fixture(scope="session")
```

Comment:
I think it's better to change the scope to module or function here. Because while I was creating tests in the other module, I got some errors here, because this fixture tries to delete the raw data after all the tests have been run but some of my tests already deleted the data.

Reply:
That's fine, I changed it earlier to session because a previous version of the memory test needed it, but that was changed and I didn't reverse this back to module. If no other tests uses it it's fine to change it to module.